src/main/java/edu/iu/terracotta/controller/app/OutcomeScoreController.java

```diff
@@ -0,0 +1,214 @@
+            } catch (DataServiceException ex) {
+                return new ResponseEntity("Unable to create outcome score: " + ex.getMessage(), HttpStatus.BAD_REQUEST);
+            }
+
+            OutcomeScore outcomeScoreSaved = outcomeScoreService.save(outcomeScore);
+            OutcomeScoreDto returnedDto = outcomeScoreService.toDto(outcomeScoreSaved);
+
+            HttpHeaders headers = new HttpHeaders();
+            headers.setLocation(ucBuilder.path("/api/experiments/{experiment_id}/exposures/{exposure_id}/outcomes/{outcome_id}/outcome_scores/{outcome_score_id}")
+                .buildAndExpand(outcomeScore.getOutcome().getExposure().getExperiment().getExperimentId(), outcomeScore.getOutcome().getExposure().getExposureId(),
+                        outcomeScore.getOutcome().getOutcomeId(), outcomeScore.getOutcomeScoreId()).toUri());
+            return new ResponseEntity<>(returnedDto, headers, HttpStatus.CREATED);
+        } else {
+            return new ResponseEntity(TextConstants.NOT_ENOUGH_PERMISSIONS, HttpStatus.UNAUTHORIZED);
+        }
+    }
+
+
+    @RequestMapping(value= "/{experiment_id}/exposures/{exposure_id}/outcomes/{outcome_id}/outcome_scores/{outcome_score_id}", method = RequestMethod.PUT)
+    public ResponseEntity<Void> updateOutcomeScore(@PathVariable("experiment_id") Long experimentId,
+                                              @PathVariable("exposure_id") Long exposureId,
+                                              @PathVariable("outcome_id") Long outcomeId,
+                                              @PathVariable("outcome_score_id") Long outcomeScoreId,
+                                              @RequestBody OutcomeScoreDto outcomeScoreDto,
+                                              HttpServletRequest req)
+            throws ExperimentNotMatchingException, OutcomeNotMatchingException, OutcomeScoreNotMatchingException, BadTokenException {
+
+        log.info("Updating outcome score with id {}", outcomeScoreId);
+        SecurityInfo securityInfo = apijwtService.extractValues(req, false);
+        apijwtService.experimentAllowed(securityInfo, experimentId);
+        apijwtService.outcomeAllowed(securityInfo, experimentId, exposureId, outcomeId);
+        apijwtService.outcomeScoreAllowed(securityInfo, outcomeId, outcomeScoreId);
+
+        if(apijwtService.isInstructorOrHigher(securityInfo)){
+            Optional<OutcomeScore> outcomeScoreSearchResult = outcomeScoreService.findById(outcomeScoreId);
+
+            if(!outcomeScoreSearchResult.isPresent()){
+                log.error("Unable to update. Outcome score with id {} not found.", outcomeScoreId);
+                return new ResponseEntity("Unable to update. Outcome score with id " + outcomeScoreId + TextConstants.NOT_FOUND_SUFFIX, HttpStatus.NOT_FOUND);
+            }
+            OutcomeScore outcomeScoreToChange = outcomeScoreSearchResult.get();
+            outcomeScoreToChange.setScoreNumeric(outcomeScoreDto.getScoreNumeric());
```

Comment:
what if they send a non numeric value?   We should be ready for it (here and surely any other place where we have numbers) and manage the exception that will happen.